genrl/deep/common/actor_critic.py

```diff
@@ -35,14 +31,76 @@ def __init__(
         hidden: Tuple = (32, 32),
         val_type: str = "V",
         discrete: bool = True,
-        **kwargs
+        **kwargs,
     ):
         super(MlpActorCritic, self).__init__()
 
         self.actor = MlpPolicy(state_dim, action_dim, hidden, discrete, **kwargs)
         self.critic = MlpValue(state_dim, action_dim, val_type, hidden, **kwargs)
 
 
+class MlpSingleActorMultiCritic(BaseActorCritic):
```

Comment:
Only Mlp?

Is there a way to do this for both? In an architecture agnostic manner?

Reply:
We don't need one for CNN as of now. 

Not sure what you mean by architecture agnostic here. You mean have MlpActorCritic be one class with options for multiple actors and critics?